backend/rates/rates.go

```diff
@@ -316,11 +328,21 @@ func (updater *RateUpdater) updateLast(ctx context.Context) {
 				updater.log.Errorf("unsupported fiat: %s", geckoFiat)
 				continue
 			}
+			if fiat == BTC.String() {
+				newVal["sat"] = rates * unitSatoshi
```

Comment:
It's better to use `sat.String()` instead of "sat"

Reply:
updated to `sat.String()`